substrate/frame/balances/src/lib.rs

```diff
@@ -227,12 +227,17 @@ pub mod pallet {
 		impl DefaultConfig for TestDefaultConfig {
 			#[inject_runtime_type]
 			type RuntimeEvent = ();
+			#[inject_runtime_type]
+			type RuntimeHoldReason = ();
```

Comment:
Huh, when we have `#[inject_runtime_type]` here, does it actually mean that the RHS here can be anything we want, and it wouldn't matter?

Reply:
It can be any type (as long as it is a valid type identifier) and any type is valid because in `DefaultConfig` it is merely a `type RuntimeHoldReason` with no bounds.